text_extensions_for_pandas/array/test_token_span.py

```diff
@@ -292,5 +294,18 @@ def test_as_frame(self):
         self.assertEqual(len(df), len(arr))
 
 
+class TokenSpanArrayIOTests(ArrayTestBase):
+
+    def test_feather(self):
+        arr = self._make_spans_of_tokens()
```

Comment:
You need to create a second test case that creates `TokenSpan`s that cover more than one token per span. It would also be a good idea to test that zero-length spans are correctly serialized/deserialized.